backend/LexBoxApi/Controllers/UserController.cs

```diff
@@ -139,16 +188,32 @@ private User CreateUserEntity(RegisterAccountInput input, LexAuthUser? jwtUser,
             Locked = false,
             CanCreateProjects = false
         };
+        UpdateUserMemberships(jwtUser, userEntity);
+        return userEntity;
+    }
+    private void UpdateUserMemberships(LexAuthUser? jwtUser, User userEntity)
+    {
         // This audience check is redundant now because of [RequireAudience(LexboxAudience.RegisterAccount, true)], but let's leave it in for safety
         if (jwtUser?.Audience == LexboxAudience.RegisterAccount && jwtUser.Projects.Length > 0)
         {
-            userEntity.Projects = jwtUser.Projects.Select(p => new ProjectUsers { Role = p.Role, ProjectId = p.ProjectId }).ToList();
```

Comment:
Would like to just do this and tell EF Core "Hey, just ignore conflicts and skip inserting that one", i.e. the Postgres query would have ON CONFLICT DO NOTHING in it. But that's not a feature available in EF Core, apparently (IIUC) because SQL Server doesn't implement the necessary feature, so EF Core can't guarantee consistency across major databases. At any rate, to avoid duplicate entries if the user already has a membership in the project, we have to replace this one-liner with the foreach loop below. In theory it's O(N^2), but that won't matter unless someone is getting invited to hundreds of projects at once, which is not going to happen.